third_party/libusb/naclport/src/libusb_over_chrome_usb.cc

```diff
@@ -523,6 +533,17 @@ int LibusbOverChromeUsb::LibusbOpen(
   if (!result.is_successful()) {
     GOOGLE_SMART_CARD_LOG_WARNING << "LibusbOverChromeUsb::LibusbOpen " <<
         "request failed: " << result.error_message();
+    // modify the devices (fake) bus number that we report so that PCSC will 
```

Comment:
tiny nit: Please capitalize the first word and end the comment with a period.

Reply:
Done